geode-serialization/src/main/java/org/apache/geode/internal/serialization/internal/DSFIDSerializerImpl.java

```diff
@@ -336,7 +336,7 @@ public void invokeFromData(Object ds, DataInput in)
 
 
   @Override
-  public void registerDSFID(int dsfid, Class<?> dsfidClass) {
+  public void register(int dsfid, Class<? extends DataSerializableFixedID> dsfidClass) {
```

Comment:
I think you should create a new unit test DSFIDSerializerImplTest. Just add test(s) for this method and focus on covering the if-else block where you added the most code.